src/module.c

```diff
@@ -3221,7 +3206,7 @@ int TagValsCommandHandler(RedisModuleCtx *ctx, RedisModuleString **argv, int arg
   }
   RS_AutoMemory(ctx);
 
-  VERIFY_ACL(ctx, argv[1], CHECK_OOM)
+  VERIFY_ACL(ctx, argv[1], IGNORE_OOM)
```

Comment:
as a general comment, since it is out of scope for the PR, I would have query validation function and call it to `VERIFY_ACL`, `handle_OOM` etc.. different PR
piggy back is abuse :)